source/Modal/Subuniverse.lagda

```diff
@@ -71,8 +71,8 @@ subuniverse-is-reflective P =
 subuniverse-is-replete
  : subuniverse 𝓤 𝓥
  → 𝓤 ⁺ ⊔ 𝓥  ̇
-subuniverse-is-replete {𝓤 = 𝓤} P =
- (A B : 𝓤 ̇)
+subuniverse-is-replete P =
+ (A B : _)
```

Comment:
`∀ A B → ...` works too fwiw

Reply:
I ban that symbol from my code, because it is mathematically weird (based on old school propositions-as-types 🧠 damage IMO)